### ISSUE TITLE
binance.WsOrderUpdate.IsInOrderBook: ReadBool: expect t or f, but found 1

### ISSUE THREAD

**Issue by f0o** [adshao/go-binance#449]
This started to happen very recently without any chances to the code. Did Binance silently update their APIs?

Full Error:
```
WS Error binance.WsOrderUpdate.IsInOrderBook: ReadBool: expect t or f, but found 1, error found in #10 byte of ...|000","W":16720467085|..., bigger context ...|.00000000","Y":"0.00000000","Q":"0.00000000","W":1672046708526,"V":"NONE"}|...
```

**Comment by f0o** [comment]
It seems that although `w` is specified as `bool` in https://binance-docs.github.io/apidocs/spot/en/#payload-order-update it might not always be transmitted?

I wonder if this error could be suppressed by `omitempty`

**Comment by f0o** [comment]
Actually this seems to be fixed in https://github.com/adshao/go-binance/pull/442 but this is not yet released...

**Comment by adshao** [comment]
> Actually this seems to be fixed in #442 but this is not yet released...

please try v2.4.0
